RMerl/asuswrt-merlin.ng
# General improvements, resolve alpha/beta comparisons

Issue originally reported for the RT-AX86U running: 388.6_alpha1-g4889cee45a
Here: https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/issues/147

# Test Environment:
1. Current production update detection script (webs_update.sh)
2. Current nvram version: 388.7_0 (or 388.7_0_rog in my case)
3. Set nvram extendno from: 0 (or 0_rog in my case) to any of the below examples.

- **Example/Test Extendno's (Not working):**
- `0_alpha1-g4889cee45a`
- `0_beta3`
- `0_Beta3`
- `Beta1`


4. Check for updates. NO UPDATES FOUND!!! When there should be.

# Next Test Environment:
1. Current production update detection script (webs_update.sh)
2. Set nvram extendno from: 0 (or 0_rog in my case) to any of the below examples.

**Example/Test Extendno (Working):**
- `beta1`
- `alpha1-g4889cee45a`


3. Check for updates. UPDATES FOUND!!! (As expected)


# Final Test Environment:
1. Patched update detection script with my version (webs_update.sh)
- `mount -o remount,rw /`
- `curl --retry 3 "https://raw.githubusercontent.com/ExtremeFiretop/asuswrt-merlin.ng-1/patch-1/release/src/router/rom/webs_scripts/merlin_webs_update.sh" -o "/usr/sbin/webs_update.sh" && chmod 755 "/usr/sbin/webs_update.sh"`

2. Set nvram extendno from: 0 (or 0_rog in my case) to any of the below examples.

**Example/Test Extendno's (NOW Working):**
- `0_alpha1-g4889cee45a`
- `0_beta3`
- `0_Beta3`
- `Beta1`

**Example/Test Extendno (Working):**
- `beta1`
- `alpha1-g4889cee45a`


3. Check for updates. UPDATES FOUND!!! (As expected). 
As you can see, this maintains the original functionality, but also extends support for the all the alpha releases, now all extendno examples are properly compared.
